<launch>
	<node pkg="push_planning" type="push_planner_node" name="push_planner_node" output="screen">
		<param name="spawn_collision_object_test" value="false"/>
	</node>

	<node pkg="push_planning" type="interactive_marker.py" name="push_planner_interaction" output="screen">
		<param name="interactive_start_state" value="false"/>
	</node>
</launch>
